k8s.gcr.io/images/k8s-staging-autoscaling/images.yaml

```diff
@@ -0,0 +1,180 @@
+    "sha256:f11ac055c20bccc488e1334554b5da6444707c2321bb811088df3bf582556f23": ["v1.3.1"]
+    "sha256:dc8b15f5d717a632492b74b935247dc0429ab4556bea5285fc15e501793f0500": ["v1.3.1-beta.1"]
+    "sha256:c15dde3d0d29c95bf828537815088e66e4b586b7ace689e6eb5eb39c75106781": ["v1.3.0"]
+    "sha256:2b845da5571475f7d5b17f32b717c148027195b768681b6c7805ed3c67f07d9d": ["v1.3.0-beta.2"]
+    "sha256:1040d210e9870178f0600d6f3bf763e936c36109c4d086724807f68d25e0c6e6": ["v1.3.0-beta.1"]
+    "sha256:b0777becbfc7a56e66b079d2767fdc173121a29165523bbbe309bcb2c0a226aa": ["v1.2.5"]
+    "sha256:d596ff09f29e7f6ff60809304dfbd03df2fc11823eae9c65c1922b97c7675166": ["v1.2.4"]
+    "sha256:91c206930916faed4984d30e2d007db182959db57c8ed47d4ef69692c85c8a16": ["v1.2.3"]  # Branches: [1.10]
+    "sha256:9aa97a73139254dcd05900ab725d275a5ddeca2e2f9d12ddca937c06798171c7": ["v1.2.3-beta.2"]
+    "sha256:c06b169a622ae75435bd41bea8420672005b1014212c110a633a873c23b43729": ["v1.2.3-beta.1"]
+    "sha256:36a369ca4643542d501bce0addf8b903f2141ae9e2608662b77a3d24f01d7780": ["v1.2.2"]
+    "sha256:d263c25904d6448e69c8b76d4514c3d5d9ec3239eeecd60b948c475934a40dba": ["v1.1.3"]  # Branches: [1.9]
+    "sha256:1e1fc9acbda4bc97ae1927723554dc8e4d2a4308080bc14f82ee174b1bd19795": ["v1.1.0"]
+    "sha256:baef01b3edc85c9ab468d76fe48ef6469f7d191506825d08d480bab5d07a8f31": ["v1.0.6"]  # Branches: [1.8]
+    "sha256:6ceb111a36020dc2124c0d7e3746088c20c7e3806a1075dd9e5fe1c42f744fff": ["v1.0.3"]
+    "sha256:657dc2ec7e02cd16ece9a80d7275023d0415820b6e9d65e5729b0b92c2ffd45a": ["v0.6.2"]  # Branches: [1.7]
+- name: cluster-proportional-autoscaler-amd64
+  dmap:
+    "sha256:a2db01cfd2ae1a16f0feef274160c659c1ac5aa433e1c514de20e334cb66c674": ["1.7.1"]
+    "sha256:c064edafc42485cd8a07ec5274c7630dcc6d05ccd8b1dbdadb1ea8f73636baa2": ["1.7.0"]
+    "sha256:0abeb6a79ad5aec10e920110446a97fb75180da8680094acb6715de62507f4b0": ["1.6.0"]
+    "sha256:85fd52e359ad2cb07dcb9863b885885ee1967bace2699f3613af06425f6611c9": ["1.5.0"]
+    "sha256:12370202895b621a2ac28226292e4578598f13c1502aa4d3ee90fff4325d9275": ["1.4.0"]
+    "sha256:4fd37c5b29a38b02c408c56254bd1a3a76f3e236610bc7a8382500bbf9ecfc76": ["1.3.0"]
+    "sha256:003f98d9f411ddfa6ff6d539196355e03ddd69fa4ed38c7ffb8fec6f729afe2d": ["1.1.2-r2"]  # Branches: ['master', 1.9, 1.8]
+    "sha256:791836c2a2471ecb23a975cf12835f2a35ada3f9d841f3f1a363b83eca99e2aa": ["1.1.2"]  # Branches: ['master', 1.9, 1.8]
+- name: cluster-proportional-autoscaler-arm
+  dmap:
+    "sha256:5dc36021546a5e81a7e6fa3c79624a4ad1ba59802739f775997b36f955046fce": ["1.7.1"]
+    "sha256:2a2b1f922861dbeb3bc9644d9fb1f42e2b2ab72fc92c1e758ca226d715fad114": ["1.6.0"]
+    "sha256:30ebd2b18e99d9c2f5b4136e38c6043c8c5c79ee6cd4827b427f623c72ac3390": ["1.5.0"]
+    "sha256:b47c34445d99635d312c5b93c7817a2f80a2cc6ab100bd5d4ab46287497e37dc": ["1.4.0"]
+    "sha256:ce21b837f64e4de2cca088d2f2318df15011cdff6b47ce416b0281040232508c": ["1.3.0"]
+- name: cluster-proportional-autoscaler-arm64
```

Comment:
I know this says autoscaler, but this is not actually currently under sig-autoscaling, we should resolve this with owners.

Reply:
I dropped the images from this PR alltogether.